src/components/wallet/views/claimableDisplay.tsx

```diff
@@ -0,0 +1,58 @@
+import { h } from '@stencil/core'
+import { has as loHas } from 'lodash-es'
+import WalletButton from './walletButton'
+import { Wallet } from '../wallet'
+
+interface ClaimableBalance {
+  id: string
+  asset: string
+  amount: string
+  sponsor?: string
+  claimants: any[]
+}
+
+export default function claimableDisplay(this: Wallet) {
+  if (this.account.availableBalances) {
+    const claimableRecords = (claimableBalances: any) => {
+      return claimableBalances.records.map(claimableBalanceRow)
+    }
+    const claimableBalanceRow = (balance: ClaimableBalance) => {
```

Comment:
Can we move this function out of the outer function for readability?

Reply:
Oh actually it looks like there are 3 function layers here, `claimableDisplay`, `claimableRecords`, and `claimableBalanceRow`. We should at least move `claimableRecords` outside `claimableDisplay`.